pkg/sql/conn_executor.go

```diff
@@ -352,6 +364,9 @@ func (s *Server) FillErrorCounts(codes, unimplemented map[string]int64) {
 	for k, v := range s.errorCounts.unimplemented {
 		unimplemented[k] = v
 	}
+	for k, v := range s.errorCounts.traces {
+		codes["internalerror-"+k] = v
```

Comment:
This isn't quite the where I was thinking: I was thinking we'd just use the general-purpose `telemetry` counters directly above instead of introducing the `traces` map at all?

 The `codes` map has error-count-by-pg-code -- adding to it will result in eventually being recorded in collected telemetry info as `errorcodes.internalerror-<k>`. That  isn't terrible, but so far, for all values of `errorcodes.<x>`, `x` was a valid pgerror code. Adding to it twould mean the sum of `errorcodes.*` would double count these since we still count them as the internalerror pg code.